featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -304,15 +308,15 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
         # ensure rows are sorted by input order
         if isinstance(feature_matrix, pd.DataFrame):
             if isinstance(cutoff_time, pd.DataFrame):
-                feature_matrix = feature_matrix.reindex(
+                feature_matrix = feature_matrix.ww.reindex(
                     pd.MultiIndex.from_frame(cutoff_time[["instance_id", "time"]],
                                              names=feature_matrix.index.names))
             else:
                 # Maintain index dtype
                 index_dtype = feature_matrix.index.get_level_values(0).dtype
-                feature_matrix = feature_matrix.reindex(cutoff_time[1].astype(index_dtype), level=0)
+                feature_matrix = feature_matrix.ww.reindex(cutoff_time[1].astype(index_dtype), level=0)
             if not cutoff_time_in_index:
-                feature_matrix.reset_index(level='time', drop=True, inplace=True)
+                feature_matrix.ww.reset_index(level='time', drop=True, inplace=True)
```

Comment:
is it ever possible that any of these reindexing could invalidate the schema by making the underlying index not match the woodwork index column?

Reply:
@thehomebrewnerd didn't you come acrosss something like this earlier today?

Reply:
I did, when I was attempting to update `_validate_cutoff_time` to retain typing information if the user already initialized WW. There is a reset index call in that function as well. It does cause the WW schema to be invalidated in some cases, which was causing at least one test failure.

Reply:
The feature matrix does not set a woodwork index so this shouldn't be an issue I believe